torch/distributed/launch.py

```diff
@@ -220,7 +201,7 @@ def main():
               "please further tune the variable for optimal performance in "
               "your application as needed. \n"
               "*****************************************".format(current_env["OMP_NUM_THREADS"]))
-
+    cmd = []
```

Comment:
consider making this a `List[(process, cmd)]` or at least a `List[cmd]`. `cmd` is double declared (once here and once in the for-loop and once outside the body of the for-loop will be equal to the last iteration's `cmd` value. If this was intended, I think there are clearer ways to represent this:

1. Using a `List` of `cmd` (as suggested in my comment)
2. Declaring a single `cmd` array before the loop but clearing it at the end of the loop
3. Declaring a `last_cmd` array before the loop and setting it equal to the local `cmd` declared within the loop at the end of the loop